### DISCUSSION TITLE
What does error P000 mean?

### DISCUSSION THREAD

**Discussion by Ein-Tim** [Digitaler-Impfnachweis/covpass-android#240] [Q&A]
Hey there! I have a user on Twitter trying to renew their certificate, but they only receive error P000. What does this error mean?

**Comment by pucherd** [comment]
Hello, R000 usually means there is an internal error occurred during reissuing process. Do you have more information about the certificate which is used?

**Reply by Ein-Tim** [reply]
I'll contact the user and let you know.

**Reply by Ein-Tim** [reply]
@pucherd [Feedback](https://social.tchncs.de/@rabln/109552275541980220):
> This is a vaccination certificate (biontech) that has expired.
Certificate issuer is the RKI. Technical expiration date was in Nov. 2022.

**Reply by Ein-Tim** [reply]
@pucherd Any chance we could get an answer on this before Christmas/new year/etc.? 

Or does this require deeper investigation. Should I advice the user to try to renew using the CWA?

**Reply by pucherd** [reply]
@Ein-Tim We are currently working on reissuing process improvements and the new version 1.39 will be released mid January. I would propose to retest it with that version.

**Reply by Ein-Tim** [reply]
@pucherd The user tried again with version 1.39.0 and it still doesn't work: https://social.tchncs.de/@rabln/109835658268659043
